query-engine/connectors/sql-connector/src/database/connector_transaction.rs

```diff
@@ -0,0 +1,51 @@
+use prisma_query::connector::Queryable;
+
+use connector_interface::*;
+use prisma_models::*;
+
+use crate::query_builder::write::NestedActions;
+use crate::query_builder::WriteQueryBuilder;
+
+pub struct ConnectorTransaction<'a> {
+    pub inner: prisma_query::connector::Transaction<'a>,
```

Comment:
We need to talk about those naming conventions in a backend meeting. I think must people are rather confused by using the exactly same term in multiple contexts.

Reply:
I made it public so i can easily call `connector_transaction.inner.commit()` in the connector impl. I will instead move that call.

Reply:
Yeah. But also you have the `prisma_query` transaction visible in one file that's not visible for people using the interface. For users of the interface, they'd see a `Transaction` from this crate and all the code using it would not use so much horizontal space.